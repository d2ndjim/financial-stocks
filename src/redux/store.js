import { configureStore } from "@reduxjs/toolkit/dist/configureStore";

import searchReducer from './search/searchSlice'
import companyReducer from './stocks/companySlice'

const store = configureStore({
  reducer: {
    search: searchReducer,
    company: companyReducer,
  },
});

export default store