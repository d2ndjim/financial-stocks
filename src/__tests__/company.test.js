import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Company from '../components/companies';

describe('Company test', () => {
  it('renders company', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Company />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
