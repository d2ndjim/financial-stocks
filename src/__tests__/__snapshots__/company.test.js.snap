// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Company test renders company 1`] = `
<div
  className="grid grid-cols-2 w-4/5 m-10 gap-4 md:grid-cols-3 md:w-4/5 md:m-auto"
/>
`;
